app/views/contexts/summary/question.py

```diff
@@ -51,17 +68,21 @@ def __init__(
             list_name=location.list_name if location else None,
             return_to=return_to,
             return_to_block_id=return_to_block_id,
+            metadata=metadata,
+            response_metadata=response_metadata,
+            language=language,
         )
 
     def get_answer(
-        self, answer_store: AnswerStore, answer_id: str
+        self, answer_store: AnswerStore, answer_id: str, list_item_id: str | None = None
     ) -> Optional[AnswerValueEscapedTypes]:
         answer = answer_store.get_answer(
-            answer_id, self.list_item_id
+            answer_id, list_item_id or self.list_item_id
         ) or self.schema.get_default_answer(answer_id)
 
         return escape_answer_value(answer.value) if answer else None
 
+    # pylint: disable=too-many-locals
```

Comment:
I'd be inclined to listen to the pylint and split this method up into some smaller methods.